nncf/internal/__init__.py

```diff
@@ -0,0 +1,12 @@
+""""
```

Comment:
Does the name `internal` mean that we have `external`? This name is not self-explainable and it is not better than `utils` or `helpers`.

Reply:
> Does the name internal mean that we have external?
Not necessarily. Whatever is in `internal` right now is `internal`. Not everything that we have outside `internal` is "not internal", but now we have a way to make the situation better.

> This name is not self-explainable
You've guessed it yourself. `internal` is as oppossed to `external`, and as an explicit marker to not use the code in this module in external user scripts, because not only there are no guarantees of compatibility, there is also no real reason to use the code here anywhere outside NNCF.

Reply:
I want to split this PR into two where the first will address the backward compatibility problem for `logging` and we can continue this discussion.
@vshampor, can you create a separate PR where logger import is placed to its previous location?

Reply:
It is already there, in #1442